test/validation/api/dma/dma.c

```diff
@@ -364,6 +366,38 @@ static void test_dma_compl_pool_max_pools(void)
 	}
 }
 
+static void test_dma_compl_user_area(void)
+{
+	odp_dma_pool_param_t dma_pool_param;
+	uint32_t size = global.dma_capa.pool.max_uarea_size;
+	odp_pool_t pool;
+	odp_dma_compl_t compl_ev;
+
+	if (size == 0) {
+		ODPH_DBG("DMA completion event user area not supported\n");
+		return;
+	}
```

Comment:
A separate condition function could be added for this test, so inactive test would be shown correctly in run summary.

Reply:
Refactored to case to be very similar to other user area tests (buffer, timer, etc.).